thirdparty/patchmatchnet/models/patchmatch.py

```diff
@@ -43,16 +43,16 @@ def forward(
 
         Args:
             random_initialization: whether to use random initialization
-            min_depth: minimum virtual depth
-            max_depth: maximum virtual depth
+            min_depth: minimum virtual depth, (B, )
+            max_depth: maximum virtual depth, (B, )
             height: height of depth map
             width: width of depth map
             depth_interval_scale: depth interval scale
             device: device on which to place tensor
-            depth: current depth
+            depth: current depth (B, 1, H', W')

```

Comment:
how is H' and W' defined?

Reply:
input `depth` will have different size when calling `patchmatch` in different stages

Reply:
it will also influence the input in `depth initialization`

Reply:
Ah I found that during the patchmatch iterations, `depth` will be None or `H'=H`